V3 board details and backlog styles

anleo opened the pull request.

anleo committed:
- added route to get specified board
- added/updated verification hasAccess to board in boardService
- added task-board resolver
- updated board resource to get specified board
- added edit board [WIP]
- updated backlog view/styles
- updated back button on task-board component
- added board title / project info on board(s) pages
- added ability (form) to add board in specified board with relations to this board

anleo committed:
- removed edit board element

anleo committed:
Merge branch 'v3' into v3-board-title-on-specific-board-page

alexkorsun commented on app/api/board/index.js:
```diff
@@ -2,6 +2,19 @@ module.exports = function (app) {
+
+        if (boardId) {
+            BoardService
+                .getById(boardId)
```
check access?

alexkorsun commented on app/services/board/BoardService.js:
```diff
@@ -47,9 +49,18 @@ let BoardService = function (Board,
     };
 
+    this.hasAccess = function(board, user) {
+        console.log('>>>>> ', board, user)
```
clean up logs

alexkorsun commented on app/services/board/BoardService.js:
```diff
@@ -47,9 +49,18 @@ let BoardService = function (Board,
         });
     };
 
+    this.hasAccess = function(board, user) {
```
maybe will better to check if user is object or id?

alexkorsun commented on public-v2/src/app/tracker/resolvers/task-board.resolver.ts:
```diff
@@ -0,0 +1,18 @@
+import {Observable} from "rxjs";
+import {BoardService} from "../services/board.service";
+import {TaskBoard} from "../models/task-board";
+import {CurrentTaskService} from "../services/current-task.service";
```
what currentTaskService for?

alexkorsun commented on public-v2/src/app/tracker/resources/board.resource.ts:
```diff
@@ -26,4 +26,9 @@ export class BoardResource extends Resource {
+  @ResourceAction({
+    url: '/api/boards/{!boardId}'
+  })
+  getBoard: ResourceMethodStrict<any, {boardId: string}, TaskBoard>;
```
use 
```
getBoard: ResourceMethod<{boardId: string}, TaskBoard>;
```

anleo committed:
- updated validation
- cleanup
- moved styles from components to stylesheet
- fixed board.project to grab its value from real rootTask instead of currentTask
